app/src/layouts/appLayout/footer/footer.scss

```diff
@@ -0,0 +1,70 @@
+.footer {
+  font-family: $FONT-REGULAR;
+  font-size: 11px;
+  line-height: 1.42857143;
+  text-align: center;
+  overflow: hidden;
+  padding: 5px 0;
+
+  &-links {
+    display: inline-block;
+    float: right;
+    padding: 0;
+    margin-right: 150px;
```

Comment:
Remove that margin. Localization switcher is temporary object. All positioning should be identical as on dev server.